VisualPinball.Unity/VisualPinball.Unity/Game/Engine/IGamelogicEngine.cs

```diff
@@ -42,7 +42,10 @@ public interface IGamelogicEngine : IGamelogicBridge
 		/// most GLEs only know about their displays when they start the game.
 		/// </remarks>
 		event EventHandler<RequestedDisplays> OnDisplaysRequested;
-		event EventHandler<DisplayFrameData> OnDisplayFrame;
+		event EventHandler<DisplayClearData> OnDisplayClear;
+		event EventHandler<DisplayFrameData> OnDisplayUpdateFrame;
+
+		void DisplayUpdateEvent(DisplayFrameData displayFrameData);

```

Comment:
I added a doc block. I'd still like the names be a little cleaner. Will work on that tonight. 

In the gles, we have 

```
		public event EventHandler<LampEventArgs> OnLampChanged;
		public event EventHandler<LampsEventArgs> OnLampsChanged;
		public event EventHandler<CoilEventArgs> OnCoilChanged;
		public event EventHandler<SwitchEventArgs2> OnSwitchChanged;
```

I'd like to have something like

```
		public event EventHandler<DisplayEventArgs> OnDisplayChanged;
```

I'm not seeing the connection with the DisplayPlayer, but I will keep looking.

Reply:
This is cleaned up, so it would be easier to re-review the file as a whole.  I added an `SetDisplay()` in the gle which gets called with an `OnDisplayChanged`. This makes it feel more like the rest of the code.